code/game/objects/items/devices/radio/radio.dm

```diff
@@ -33,6 +33,9 @@
 	var/use_command = FALSE  // If true, broadcasts will be large and BOLD.
 	var/command = FALSE  // If true, use_command can be toggled at will.
 
+	///makes anyone who is talking through this anonymous.
+	var/anonimize = FALSE
```

Comment:
``anonymize`` is the correct spelling, can you fix this

Reply:
yeah